crates/holochain/src/conductor/state.rs

```diff
@@ -7,9 +7,22 @@ use holochain_types::prelude::*;
 use serde::Deserialize;
 use serde::Serialize;
 use std::collections::HashMap;
+use std::sync::Arc;
 
 use super::error::{ConductorError, ConductorResult};
 
+/// Unique conductor tag / identifier.
+#[derive(Clone, Deserialize, Serialize, Debug, SerializedBytes)]
+#[cfg_attr(test, derive(PartialEq))]
+#[serde(transparent)]
+pub struct Tag(pub Arc<str>);
```

Comment:
Could you just name this something like `ConductorTag` or `ConductorStateTag` so it's clearly differentiated from `LinkTag`?

```suggestion
pub struct ConductorStateTag(pub Arc<str>);
```